pkgs/development/node-packages/default-v10.nix

```diff
@@ -116,4 +116,6 @@ nodePackages // {
   thelounge = nodePackages.thelounge.override {
     buildInputs = [ nodePackages.node-pre-gyp ];
   };
+
+  three = nodePackages."three-0.108.0".override {};
```

Comment:
You could probably apply a patch fixing the issue instead of pinning:
```
three = nodePackages.three.overrideAttrs(old: {
  patches = [ (fetchpatch {url="https://something.com/patch", sha256 = "....";}) ];
})
```